twisted/towncrier
# #672 Add documentation to the dist.

# Description

Fixes #672


Update the pyproject.toml

# How to test

Run `python -m build`.
It will generate  `.tar.gz` and `.whl` files inside the `dist/` folder.
Check that the documentation source files are included in the `.tar.gz` version.
Check that the documentation files are not included in the `.whl` version.

`.whl` is just a zip file.
